docs/examples/statistic/countdown.vue

```diff
@@ -3,28 +3,28 @@
     <div class="s-card">
       <el-row>
         <el-col :span="8">
-          <ElCountdown title="开始抢购" countdown :value="value1" />
+          <ElCountdown title="Start to grab" countdown :value="value" />
         </el-col>
         <el-col :span="8">
-          <ElCountdown title="剩余VIP时长" format="HH:mm:ss" :value="value3" />
+          <ElCountdown
+            title="Remaining VIP time"
+            format="HH:mm:ss"
+            :value="value1"
+          />
           <div class="f-center">
-            <el-button type="primary" @click="reset">重置</el-button>
+            <el-button type="primary" @click="reset">Reset</el-button>
           </div>
         </el-col>
         <el-col :span="8">
-          <ElCountdown
-            :value="value4"
-            countdown
-            format="DD 天 HH 时 mm 分 ss 秒"
-          >
+          <ElCountdown :value="value2" format="DD HH : mm : ss ">
```

Comment:
```suggestion
          <ElCountdown :value="value2" format="DD Days HH:mm:ss ">
```
No need to use space between date.